### PR TITLE
Added unit test for MultipleChoiceItem

### LINKED ISSUE #2530: [Code health] Add Unit Test MultipleChoiceItem
Add Unit Test MultipleChoiceItem

### PR DESCRIPTION
**Pull request by anandwana001**
Fixes #2530 

Screenshot 2024-07-02 at 09 23 02

@shobhitagarwal1612 @sufyanAbbasi  PTAL?

### CODE REVIEW

File: ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceItemTest.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      code = "code",
+      label = "label"
+    ),
+    isSelected = true,
+    cardinality = Cardinality.SELECT_MULTIPLE
+  )
+
+
+  @Test
+  fun testIsTheSameItemTrue() {
```
Comment (sufyanAbbasi) [inline ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceItemTest.kt:62]: Use string function name for tests, e.g. 
```
fun `isTheSameItem returns true when IDs are the same`()
```

See: https://github.com/google/ground-android/blob/a410aff6e2ea2f5f44a3a52418515507d9b44e61/ground/src/test/java/com/google/android/ground/model/task/ConditionTest.kt#L56

Also below.

**Review by sufyanAbbasi — CHANGES_REQUESTED** [review]
Thank you so much! Just a few quick changes.

File: ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceItemTest.kt (hunk 1 above)
Comment (anandwana001) [reply ground/src/test/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceItemTest.kt]: Done

**Review by sufyanAbbasi — APPROVED** [review]
Fantastic! Thank you!

**Comment by sufyanAbbasi** [comment]
Sorry for missing this earlier, looks like there's a formatting error we can quickly resolve by running `ktfmtFormat`:

 > [ktfmt] Found 1 files that are not properly formatted:
src/test/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceItemTest.kt

I can do that right now

**Pull request merged**